src/spikeinterface/sortingcomponents/clustering/circus.py

```diff
@@ -152,9 +153,9 @@ def main_function(cls, recording, peaks, params):
                 sub_data = sub_data.reshape(len(sub_data), -1)
 
                 if all_pc_data.shape[1] > params["n_svd"][1]:
-                    tsvd = TruncatedSVD(params["n_svd"][1])
+                    tsvd = PCA(params["n_svd"][1], whiten=True)
```

Comment:
For my own learning why should you whiten here if you've already whitened the recording above?

Reply:
The recording has been whiten, but it can not hurt to rewhiten the individual waveforms snippets that have been collected before clustering.

Reply:
I wasn't sure if this was something similar to if you double filter a recording you actually change the order of the filter, which may not necessarily be a bad thing. thanks Pierre.